includes/data-events/connectors/ga4/class-ga4.php

```diff
@@ -412,14 +406,37 @@ private static function send_event( Event $event, $client_id, $timestamp, $user_
 			$payload['user_id'] = $user_id;
 		}
 
-		wp_remote_post(
-			$url,
-			[
-				'body' => wp_json_encode( $payload ),
-			]
-		);
+		$destinations = [
+			self::get_credentials(),
+		];
```

Comment:
Shouldn't this array start empty and only include the local credentials if they are filled?

Reply:
> Shouldn't this array start empty and only include the local credentials if they are filled?

This is basically what happens. `get_credentials` will return an array with empty values.

> Also, what do you think of calling it $properties?

Agree. Changed in https://github.com/Automattic/newspack-plugin/pull/2503/commits/7cdf13a892053973f80dc1868a98ad2e92733ec9

Also updated the README and changed the filter name on the manager plugin -> https://github.com/Automattic/newspack-manager/pull/128/commits/83d82123e5a491e7b9b540d0c39f24a3d1950ffd